packages/preload/src/services/index.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'node:path';
+
+import * as config from './config';
+import * as fs from './fs';
+import * as npm from './npm';
+import * as ipc from './ipc';
+import * as pkg from './pkg';
+
+export type Services = {
+  config: typeof config;
+  fs: typeof fs;
+  npm: typeof npm;
+  ipc: typeof ipc;
+  path: typeof path;
+  pkg: typeof pkg;
+};
+
+export const getServices = (): Services => ({
```

Comment:
this is what i want to remove, we are doing an `import * from`

dont we have an init function where it all starts to create each component there and pass only that reference to all the other files ?

Reply:
@gonpombo8 do you mean something similar to the `initializeWorkspace`?